package.json

```diff
@@ -65,6 +67,9 @@
     "element-collapse": "1.1.0",
     "esbuild": "0.20.1",
     "eslint": "8.56.0",
+    "@warp-ds/eslint-config": "1.0.4",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-config-prettier": "^9.1.0",
```

Comment:
Could you also pin these dependencies so renovate bot manages to update them here when the time comes? 🙏

Reply:
Updated! :)